test/plz_run/BUILD

```diff
@@ -58,3 +58,13 @@ please_repo_e2e_test(
     plz_command = "plz test //src:_add_data_file_and_target",
     repo = "test_repo",
 )
+
+# Test --wd flag changes directory and outputs there
+please_repo_e2e_test(
+    name = "test_wd_flag",
+    expected_output = {
+        "test_wd/guava.txt": "guava",
+    },
+    plz_command = "mkdir test_wd; plz run --wd test_wd //src:create_file",
```

Comment:
Can we add a test to check that `--wd=.` works the same as the old `--in_wd`?

Reply:
Added another test. Seems to just work!